editor/src/components/editor/url-utils.ts

```diff
@@ -0,0 +1,7 @@
+export function readParamFromUrl(param: string): string | null {
```

Comment:
can we explicitly state that this is not only reading the param from the URL but also removing it from the current window location, since this function call is not idempotent? (or just do the reading in this function and do the window replacement at the caller's site)